src/api/services/history/purchaseHistory.ts

```
@@ -0,0 +1,11 @@
+import axiosInstance from '@/api/client/axios';
+import { PurchaseHistoryResponse } from '@/api/types/history';
+
+export const purchaseHistory = async (): Promise<PurchaseHistoryResponse[] | undefined> => {
+  try {
+    const response = await axiosInstance.get('/v1/mypage/purchase-histories');
+    return response.data.content.purchaseHistoriesRes;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)